src/style.css

```
@@ -0,0 +1,121 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.page {
+  box-sizing: border-box;
+  width: 100%;
+  border-radius: 0;
+  margin: 0 auto;
+  background: #fff;
+  position: relative;
+}
+
+.bigForm {
+  height: 70px;
+  margin: 20px 0;
+}
+
+.bigField {
+  position: relative;
+}
+
+.bigBar {
+  box-sizing: border-box;
+  width: 100%;
+  height: 100%;
+  background: #fff;
+  font-family: Avenir, sans-serif;
```

move font-family to the `html` tag selector and let inputs inherit it 
```suggestion
  font-family: inherit;
```